website/source/docs/platform/k8s/run.html.md

```diff
@@ -43,59 +46,122 @@ the [recommended security features](/docs/internals/security.html). Currently,
 some of these features are not supported in the Helm chart and require additional
 manual configuration.
 
-## How-To
+### Prerequisites
+
+The Consul Helm chart works with Helm 2 and Helm 3. If using Helm 2, you will
+need to install Tiller by following the
+[Helm 2 Installation Guide](https://v2.helm.sh/docs/using_helm/#quickstart-guide).
 
 ### Installing Consul
 
-To install Consul, clone the consul-helm repository, checkout the latest release, and install
-Consul. You can run `helm install` with the `--dry-run` flag to see the
-resources it would configure. In a production environment, you should always
-use the `--dry-run` flag prior to making any changes to the Consul cluster
-via Helm.
+Determine the latest version of the Consul Helm chart
+by visiting [https://github.com/hashicorp/consul-helm/releases](https://github.com/hashicorp/consul-helm/releases).
 
-```sh
-# Clone the chart repo
-$ git clone https://github.com/hashicorp/consul-helm.git
-$ cd consul-helm
+Clone the chart at that version, for example if the latest version is
+`v0.8.1` you would run:
 
-# Checkout a tagged version
-$ git checkout v0.1.0
+```bash
+$ git clone --single-branch --branch v0.8.1 https://github.com/hashicorp/consul-helm.git
+Cloning into 'consul-helm'...
+...
+You are in 'detached HEAD' state...
+```
+
+Ensure you've checked out the correct version with `helm inspect chart`:
 
-# Run Helm
-$ helm install --name consul ./
+```bash
+$ helm inspect chart ./consul-helm
+apiVersion: v1
+description: Install and configure Consul on Kubernetes.
+home: https://www.consul.io
+name: consul
+sources:
+- https://github.com/hashicorp/consul
+- https://github.com/hashicorp/consul-helm
+- https://github.com/hashicorp/consul-k8s
+version: 0.8.1
+```
+
+Now you're ready to install Consul! To install Consul with the default
+configuration run:
+
+```sh
+$ helm install hashicorp ./consul-helm
+NAME: hashicorp
 ...
```

Comment:
It'd be nice to say here explicitly that these instructions are for Helm 3. I feel like Helm 3 hasn't become the de facto standard yet and people are still transitioning to it.

Reply:
Done.